### PR TITLE
[DOC-192] [2.20.2] Backup portability (Enable backup sets to be copied or moved)

### PR DESCRIPTION
**Pull request by aishwarya24**
https://deploy-preview-20980--infallible-bardeen-164bc9.netlify.app/stable/yugabyte-platform/back-up-restore-universes/back-up-universe-data/#access-backups-in-storage

### CODE REVIEW

File: docs/content/stable/yugabyte-platform/back-up-restore-universes/back-up-universe-data.md [hunk 1]
```diff
@@ -136,8 +138,50 @@ s3://user_bucket/some/sub/folders
 
 A backup set consists of a successful full backup, and (if incremental backups were taken) one or more consecutive successful incremental backups. The backup set can be used to restore a database at the point in time of the full and/or incremental backup, as long as the chain of good incremental backups is unbroken. Use the creation time to identify increments that occurred after a full backup.
 
-Although you can move a backup from its location, for a successful restore, none of the sub-components and folder names can be modified (from the universe UUID folder on down).
+Although you can move a backup from its location, for a successful restore, only the bucket can be changed, and none of the sub-components and folder names can be modified (from the sub-directories on down). For examples, see [Moving backups between buckets](#moving-backups-between-buckets).
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/back-up-restore-universes/back-up-universe-data.md:141]: would move this par to new moving backups section as intro

Comment (aishwarya24) [reply docs/content/stable/yugabyte-platform/back-up-restore-universes/back-up-universe-data.md:141]: Had it this way originally. will revert the change.

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data.md [hunk 2]
```diff
@@ -108,22 +108,34 @@ To view the details of a restored database, navigate to **Universes > Restore Hi
 
 ## Advanced restore procedure
 
-In addition to the basic restore, an advanced option is available for when you have more than one YugabyteDB Anywhere installation and want to restore a database or keyspace from a different YugabyteDB Anywhere installation to the current universe.
+In addition to the basic restore, an advanced backup option is available for the following circumstances:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data.md:111]:
```suggestion
In addition to the basic restore, an advanced restore option is available for the following circumstances:
```

File: docs/content/stable/yugabyte-platform/back-up-restore-universes/restore-universe-data.md [hunk 3]
```diff
@@ -105,22 +105,34 @@ To view the details of a restored database, navigate to **Universes > Restore Hi
 
 ## Advanced restore procedure
 
-In addition to the basic restore, an advanced option is available for when you have more than one YugabyteDB Anywhere installation and want to restore a database or keyspace from a different YugabyteDB Anywhere installation to the current universe.
+In addition to the basic restore, an advanced backup option is available for the following circumstances:
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/back-up-restore-universes/restore-universe-data.md:108]:
```suggestion
In addition to the basic restore, an advanced restore option is available for the following circumstances:
```

**Pull request merged**